File: python/calculator.py
i=0
while i<10:

    print("\t""\t""OUR CALCULATOR")
    print(" 1.) ADDITION""\n"" 2.) SUBTRACTION""\n"" 3.) DIVISION""\n"" 4.) MULTIPLICATION""\n"" 5.) Print Table""\n")
    A = int(input("choose one:"))
    if A==1:
        a=float(input("Enter value :"))
        b=float(input("Enter value :"))
        c=a+b
        print(c)

    elif A==2:
        a=float(input("Enter value :"))
        b=float(input("Enter value :"))
        c=a-b
        print(c)

    elif A==3:
        a=float(input("Enter value :"))
        b=float(input("Enter value :"))
        c=a/b
        print(c)

    elif A==4:
        a=float(input("Enter value :"))
        b=float(input("Enter value :"))
        c=a*b
        print(c)

    elif A==5:
        num =int(input("enter number for table :"))
        for i in range(1,11):
            print (str(num) + " X " + str(i) + " = " + str(i*num))
    

    else:
        print("not in a list ")

    

File: c/calculator.c
#include<stdio.h>
int main(){
    int n;
    float a,b;
    printf("***MENU***\n");
    printf("1.) Addition\n 2.) Substraction\n 3.) Division\n 4.) Multiplication\n");
    printf("Choose an operation serial no. as your wish:\n");
    scanf("%d",&n);
    switch (n) {
    case 1:
    printf("enter number:");
    scanf("%f",&a);
    printf("enter number:");
    scanf("%f",&b);
   float sum=(a+b);
    printf("Addition is %f",sum);
    break;
    case 2:
    printf("enter number:");
    scanf("%f",&a);
    printf("enter number:");
    scanf("%f",&b);
   float sub =(a-b);
    printf("Subtraction is %f",sub);
    break;
    case 3:
    printf("enter number:");
    scanf("%f",&a);
    printf("enter number:");
    scanf("%f",&b);
   float mul=(a*b);
    printf("Multiplication is %f",mul);
    break;
    case 4:
    printf("enter number:");
    scanf("%f",&a);
    printf("enter number:");
    scanf("%f",&b);
  float div =(a/b);
    printf("Division is %f",div);
    break;
 default :
printf("not in listed ");
    break;
}
return 0;
}
